### PR TITLE
Sprinter atmos rework

### PR DESCRIPTION
**Pull request by Eagle0600**
## About the PR
Added fore and aft air alarm.
Added vent and scrubber to bar.
Linked atmos devices to air alarms.

Reevaluated shuttle price.

Changed medbay airlock to be appropriately medical.
Replaced sodium lights with fluorescent lights.

## Why / Balance
Adding the air alarms and linking the devices brings the shuttle in line with current guidelines.
The shuttle's value has fallen since its last appraisal. Adding a 30% markup brings it in line with other expedition shuttles.
The medbay airlock should be properly marked.
This is a shuttle with a central bar. People are intended to be able to relax here, even if it is a bit grungy. Sodium lights run counter to this and are, frankly, annoying as a player to have ship-wide.

## How to test
Purchase the Sprinter. Walk around. Play with the atmos alarms to check they function (fore alarm in engineering, aft alarm in the main hallway). Check that the vents and scrubbers all have expected gas levels after a short while to ensure they are properly connected to the distribution and waste networks.

### Air Alarms
airalarmaft
airalarmfore

### Medbay
medbay

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

File: Resources/Maps/_NF/Shuttles/Expedition/sprinter.yml
Comment (dvir001) [inline Resources/Maps/_NF/Shuttles/Expedition/sprinter.yml:4100]: Ah

Comment (Eagle0600) [reply Resources/Maps/_NF/Shuttles/Expedition/sprinter.yml:4100]: Is this a problem? I did explain why I was doing it in the PR.

**Comment by Eagle0600** [comment]
That shipyard test failure is based on the phoenix (apparently it can be sold for more than the current price), which is of no concern to this PR.

**Review by whatston3 — APPROVED** [review]
Looks fine, air alarms seem well linked, YAML changes are minimal, approving, would recommend merging with a review from another pair of eyes.  Will leave this open for a few days pending a separate approval, but looks good to go from my side.

For what it's worth, in-game appraisal via shipyard resale shows resale values at ~44.5k at the moment.

**Pull request merged**
